src/huggingface_hub/repocard.py

```diff
@@ -463,6 +463,17 @@ def from_template(  # type: ignore # violates Liskov property but easier to use
         return super().from_template(card_data, template_path, **template_kwargs)
 
 
+class SpaceCard:
+    """Space card is an alias for [`RepoCard`].
+
+    At the moment, it does not implement any specific logic. `SpaceCard` is defined for
```

Comment:
Do you think it makes sense to have Space-specific attributes in `RepoCard` under this instead? (e.g. like the one that tells us about duplicate)

Reply:
yes!!

Reply:
Defintely. Not useless in the end :smile:

Reply:
@merveenoyan @julien-c added it in https://github.com/huggingface/huggingface_hub/pull/1354/commits/d5e71bb62e000acd2d4a63ad2067eba2b8526b4d